powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/CredentialCounterService.java

```diff
@@ -137,14 +137,26 @@ public void updateCredentialCounter(CredentialEntity credential, AuthenticationR
     @Transactional
     public ResetCountersResponse resetCounters(ResetCountersRequest request) {
         final List<CredentialEntity> blockedCredentials = credentialRepository.findAllByStatus(CredentialStatus.BLOCKED_TEMPORARY);
```

Comment:
Consider paging.

Reply:
I will switch to SQL update to avoid performance issues.